diff --git a/include/configs/sunxi-common.h b/include/configs/sunxi-common.h
index 02d7be0..6edca77 100644
--- a/include/configs/sunxi-common.h
+++ b/include/configs/sunxi-common.h
@@ -485,7 +485,7 @@ extern int soft_i2c_gpio_scl;
 	CONSOLE_STDOUT_SETTINGS
 
 #ifdef CONFIG_ARM64
-#define FDTFILE "allwinner/" CONFIG_DEFAULT_DEVICE_TREE ".dtb"
+#define FDTFILE CONFIG_DEFAULT_DEVICE_TREE ".dtb"
 #else
 #define FDTFILE CONFIG_DEFAULT_DEVICE_TREE ".dtb"
 #endif
